lib/customer.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+require 'csv'
+
+class Customer
+  attr_reader(:id)
+  attr_accessor(:email, :address)
```

Comment:
Do email and address need to be accessors?  Default to providing users of your class the minimum access they need to work with it, it will limit opportunities for them to use your class incorrectly. 

These should be `attr_reader` methods.